packages/ui/react-ui-navtree/src/components/NavTreeItem.tsx

```diff
@@ -158,6 +164,7 @@ const NavTreeItemImpl = forwardRef<HTMLDivElement, MosaicTileComponentProps<NavT
             setMenuOpen(true);
           }}
           {...draggableProps}
+          aria-describedby=''
```

Comment:
what's the significance of this?

Reply:
I wanted to leave a comment about this but couldn’t find a way that our estlint rules would allow… basically `draggableProps` includes an aria description with instructions on how to drag the item using the keyboard, but that is not supported here for now.

I think I’d notice this when returning to fix the DnD issues here, but if you have any recommendations on how to annotate this I’d be happy to leave an annotation.

Reply:
Figured it out